src/commands/referral/referralNew.test.ts

```diff
@@ -0,0 +1,170 @@
+import { autocomplete, execute } from './referralNew';
+import { getPrismaClient } from '../../clients';
+import { services } from './services';
+
+jest.mock('../../clients');
+const mockGetPrismaClient = jest.mocked(getPrismaClient);
+
+describe('autocomplete', () => {
+  it('should return nothing if the search term shorter than 4', () => {
+    autocomplete({
+      options: {
+        getString: (name: string, required: true) => {
```

Comment:
cái test của `getString` với `respond` có cách nào move nó ra khỏi cái call của `autocomplete` đc ko? I mean it works, but để đây hơi khó nhìn where stuff is actually asserted.

Maybe tạo 1 cái `createAutocompleteInteraction` ở trên cùng, lấy 2 param `getString` + `respond` là 2 cái callback rồi fill vào?